app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -1,7 +1,7 @@
 package org.systers.mentorship.view.activities
 
 import androidx.lifecycle.Observer
-import androidx.lifecycle.ViewModelProviders
+import androidx.lifecycle.ViewModelProvider
```

Comment:
can you use the by viewModels delegator?

Reply:
Isn't this working ?

Reply:
It does work however if even this is deprecated we won't need to make a change in the future because the delegator always uses the best and upto date method.

Reply:
Okay i will update the code